### PR TITLE
Presentation

### PR DESCRIPTION
**Pull request by Cindie1908**
Branch with all Users Stories presented last Friday.

### CODE REVIEW

File: src/components/LogicModal.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    constructor() {
+        this.state = {
+            revele: false,
+        }
+    }
+
+    toggleModal() {
+        this.setState({revele: !this.state.revele})
+    }
+}
```
Comment (erabin05) [inline src/components/LogicModal.js]: remove this copy of the component above

**Review by erabin05 — CHANGES_REQUESTED** [review]
Ok, exept for one componenet who is repeated twice in the same file in ```logicModal.js```

File: src/components/LogicModal.js (hunk 1 above)
Comment (Cindie1908) [reply src/components/LogicModal.js]: It's good !

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**